src/from_markdown.js

```diff
@@ -81,8 +81,8 @@ class MarkdownParseState {
   }
 }
 
-function attrs(given, token) {
-  return given instanceof Function ? given(token) : given
```

Comment:
@marijnh yes, I'm trying to match it with `ParseRule`. Added `instanceof` check.